kotlin/samples/hello-terminal/hello-terminal-app/src/main/java/com/squareup/sample/helloterminal/HelloWorkflow.kt

```diff
@@ -0,0 +1,81 @@
+/*
+ * Copyright 2019 Square Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.squareup.sample.helloterminal
+
+import com.squareup.sample.helloterminal.HelloWorkflow.State
+import com.squareup.sample.helloterminal.terminalworkflow.ExitCode
+import com.squareup.sample.helloterminal.terminalworkflow.KeyStroke.KeyType.Backspace
+import com.squareup.sample.helloterminal.terminalworkflow.TerminalRendering
+import com.squareup.sample.helloterminal.terminalworkflow.TerminalRendering.Color.GREEN
+import com.squareup.sample.helloterminal.terminalworkflow.TerminalSize
+import com.squareup.sample.helloterminal.terminalworkflow.TerminalWorkflow
+import com.squareup.workflow.RenderContext
+import com.squareup.workflow.Snapshot
+import com.squareup.workflow.StatefulWorkflow
+import com.squareup.workflow.WorkflowAction.Companion.emitOutput
+import com.squareup.workflow.WorkflowAction.Companion.enterState
+import com.squareup.workflow.WorkflowAction.Companion.noop
+import com.squareup.workflow.renderChild
+
+class HelloWorkflow : TerminalWorkflow,
```

Comment:
Still don't think this class should be called `HelloWorkflow`. I doubled-up that name in the other two samples b/c it's copy/paste identical, and really is a Hello World type of thing. I'd rather that's what gets found when people search for `HelloWorkflow`. 

`HelloTerminalWorkflow`? `SimpleTerminalWorkflow`? `TerminalWorkflowImpl`? `RealTerminalWorkflow`?

Reply:
`HelloTerminalWorkflow` it is.